build/ros3d.js

```diff
@@ -328,9 +331,9 @@ ROS3D.DepthCloud = function(options) {
     '    float z = -depth;',
     '    ',
     '    pos = vec4(',
-    '      ( position.x / width - 0.5 ) * z * (1000.0/focallength) * -1.0,',
-    '      ( position.y / height - 0.5 ) * z * (1000.0/focallength),',
-    '      (- z + zOffset / 1000.0) * 2.0,',
+    '      ( position.x / width - 0.5 ) * z * 0.5 * maxDepthPerTile * (1000.0/focallength) * -1.0,',
+    '      ( position.y / height - 0.5 ) * z * 0.5 * maxDepthPerTile * (1000.0/focallength),',
+    '      (- z + zOffset / 1000.0) * maxDepthPerTile,',
```

Comment:
It looks like it changes the default behavior with default value of `maxDepthPerTile`. Can you make the default behavior as before?